default/python/test/AI/Behavior/freeOrionAIInterface.py

```diff
@@ -0,0 +1,20 @@
+# Fake fo for testing Behavior purposes
+
+# TODO Find a better way to import the static portions of fo without the test chaos, overhead and random state of fo.
```

Comment:
I tried to take `agression, userString and userStringList` from where you indicated.  It only works if I have empty __init__.py files in ./freeorion_debug/ and ./freeorion_debug/ide_tools/results/.  The imports in ./freeorion_debug/__init__.py assume that a running freeorion session exists.  I made a comment in the code, but did not change the __init__.py file, because that will break the interactive debugger.

Once the imports are sorted out it works perfectly.

If you have some insight in how to fix it then I will fix it.  Otherwise, I'll leave it as is and fix it after I understand how the interactive portion of the freeorion_debug module works.